docs/docs/usage/scaladoc/scaladocDocstrings.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Scaladoc docstrings - specific Tags and Features
+---
+
+# {{page.title}}
+
+This chapter describes how to correctly write docstrings and how to use all the available features of scaladoc.
+Since many things are the same as in the old scaladoc, some parts are reused from this [article](https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html)
+
+
+Scaladoc extends Markdown with additional features, such as linking
+to API definitions. This can be used from within static documentation and blog
+posts to provide blend-in content.
+
+
+## Where to put docstrings
+
+Scaladoc comments go before the items they pertain to in a special comment block that starts with a /** and ends with a */, like this:
+
+```scala
+/** Start the comment here
+  * and use the left star followed by a
+  * white space on every line.
+  *
+  * Even on empty paragraph-break lines.
+  *
+  * Note that the * on each line is aligned
+  * with the second * in /** so that the
+  * left margin is on the same column on the
+  * first line and on subsequent ones.
+  *
+  * Close the comment with *\/
+  *
+  * If you use Scaladoc tags (@param, @group, etc.),
+  * remember to put them at separate lines with nothing preceding.
+  *
+  * For example:
+  *
+  * Calculate the square of the given number
+  *
+  * @param d the Double to square
+  * @return the result of squaring d
+  */
+ def square(d: Double): Double = d * d
+```
+
+In the example above, this Scaladoc comment is associated with the method square since it is right before it in the source code.
+
+Scaladoc comments can go before fields, methods, classes, traits, objects and even (especially) package objects. Scaladoc comments for package objects make a great place to put an overview of a specific package or API.
```

Comment:
There are no package objects anymore in Scala 3? How do we document a package?

Reply:
There's an [issue](https://github.com/lampepfl/dotty/issues/11284) to deal with this problem, however many intuitive solutions require changes in sbt

Reply:
So, should we change/remove the part about package objects?

Reply:
This is some copy-paste leftover, I'll change it and point it to the github issue.